src/altair/config.py

```diff
@@ -3,6 +3,7 @@
 
 class Config:
     TELEGRAM_WEBHOOK_URL: str = os.environ.get("TELEGRAM_WEBHOOK_URL")
+    APPROVED_USER: str = "444943133" # Matt - this is your user ID from Telegram.
```

Comment:
I know I mentioned this already but just doing it the GitHub way. This would be better as an environmental variable so it can be configurable.

Reply:
Sure, I can do that. Should I add this in the Fly.toml?

Reply:
Fixed.

Reply:
A list of variables needed to configure it should go into the README really. But I haven't exactly started that trend and it doesn't seem fair to ask you to fix all of that. I'm fine not documenting it for now.

Reply:
Agreed. You'll notice I used a default value for the `APPROVED_USER` environ. I would also use a default empty string for the rest of your ENVs, or a default string, as currently they can return one of two types.

Or you could wrap those environ calls in a try/catch?